src/Word/Helpers.elm

```diff
@@ -40,37 +40,6 @@ safeShiftRightZfBy n val =
         Bitwise.shiftRightZfBy n val
 
 
-{-| Unsigned shift left by `n`.
-
-    safeShiftLeftBy 1 0x7FFFFFFF |> Hex.fromInt 8
-    --> "fffffffe"
-
-    safeShiftLeftBy 1 0x3FFFFFFF |> Hex.fromInt 8
-    --> "7ffffffe"
-
-    safeShiftLeftBy 2 0x3FFFFFFF |> Hex.fromInt 8
-    --> "fffffffc"
-
-    safeShiftLeftBy 31 0xFFFFFFFF |> Hex.fromInt 8
-    --> "80000000"
-
--}
-safeShiftLeftBy : Int -> Int -> Int
-safeShiftLeftBy n val =
-    Bitwise.shiftLeftBy n val
-        % 0x0000000100000000
```

Comment:
On the windows elm repl `0x0000000100000000` evaluates to `0` which was causing the modulo by zero errors. Turns out that these "safe" operations are not necessary.